src/Contrib/Zipkin/SpanConverter.php

```diff
@@ -98,13 +111,14 @@ public function convert(SpanDataInterface $span)
         }
 
         foreach ($span->getEvents() as $event) {
-            if (!array_key_exists('annotations', $row)) {
-                $row['annotations'] = [];
+            $row['annotations'][] = SpanConverter::toAnnotation($event);
+        }
+
+        if (($span->getKind() === SpanKind::KIND_CLIENT) || ($span->getKind() === SpanKind::KIND_PRODUCER)) {
+            $remoteEndpointData = SpanConverter::toRemoteEndpoint($span);
+            if ($remoteEndpointData != null) {
```

Comment:
Did you use a weak comparison on purpose?

Reply:
Nope, nice catch!